### ISSUE TITLE
More results for Linear SRA?

### ISSUE THREAD

**Issue by kpmokpmo** [whai362/PVT#50]
Hi, it seems Linear SRA works better with fewer params on PVT-V2-B2. Could you please show more results when applying this Attention to other model variants?

**Comment by whai362** [comment]
Yes, linear SRA will be used for all variants. We are making more improvements to linear SRA. All models will be released later.
Thanks for your attention.

**Comment by cuixing61** [comment]
hi, when i tried to train the model by myself, lots of warning appeared.
UserWarning: Argument interpolation should be of type InterpolationMode instead of int. Please, use InterpolationMode enum.
Warning: Leaking Caffe2 thread-pool after fork. (function pthreadpool)
although these warnings seems no harm when training the model

**Comment by xingshulicc** [comment]
The PVTv2 used the Patch Merging (PM) or not?
I can not see the implementation of PM. Can you explain that to me, thank you.

**Comment by OliverRensu** [comment]
Hi, do you have more results of the linear SRA?  We want to compare them in our paper. Thanks~
